lib/rspec/mocks/error_generator.rb

```diff
@@ -268,6 +268,17 @@ def unexpected_arguments_message(expected_args_string, actual_args_string)
       def error_message(expectation, args_for_multiple_calls)
         expected_args = format_args(expectation.expected_args)
         actual_args = format_received_args(args_for_multiple_calls)
+
+        if RUBY_VERSION >= "3" && RSpec::Support::RubyFeatures.kw_args_supported? && expected_args == actual_args
```

Comment:
Cosmetic: with `RUBY_VERSION >= "3"`, I believe `RSpec::Support::RubyFeatures.kw_args_supported?` will always evaluate to `true`, so it's redundant.

Reply:
Well, `ruby2_keywords` might go away one day (not any time soon though).

Reply:
[`kw_args_supported?`](https://github.com/rspec/rspec-support/blob/623608b4f49c063fca2616cb3e35ae62ea5fe5cf/lib/rspec/support/ruby_features.rb#L130) has no awareness of that.
Currently, this code expands to:
```ruby
if RUBY_VERSION >= "3" && RUBY_VERSION >= '2.0.0'
```